hspec-core/src/Test/Hspec/Core/Formatters/Pretty/Unicode.hs

```diff
@@ -0,0 +1,28 @@
+module Test.Hspec.Core.Formatters.Pretty.Unicode (
+  ushow
+, ushows
+) where
+
+import           Prelude ()
+import           Test.Hspec.Core.Compat
+
+import           Data.Char
+
+ushow :: String -> String
+ushow xs = ushows xs ""
```

Comment:
@igrep @ncaq if you only care about `String`s, `ushow` can be implemented like this (which is simpler and probably faster).

Reply:
Thank you. Fantastic!